packages/bpk-component-close-button/src/bpk-close-button.scss

```diff
@@ -26,10 +26,10 @@
   appearance: none;
 
   &__icon {
-    fill: $bpk-modal-close-icon-fill;
+    @include bpk-themeable-property(fill, --bpk-modal-close-button-color, $bpk-modal-close-icon-fill);
 
     @include bpk-hover {
-      fill: $bpk-modal-close-icon-hover-fill;
+      @include bpk-themeable-property(fill, --bpk-modal-close-button--hover-color, $bpk-modal-close-icon-hover-fill);
```

Comment:
same as above, name can be changed

Reply:
should just be `--bpk-modal-close-button-hover-color` :)